datalake/backend/tests/conftest.py

```diff
@@ -12,6 +13,16 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture()
+def batch_client():
+    return boto3.client("batch")
```

Comment:
I think this test might belong more in the root `tests` directory:

- It's an infrastructure test, so it requires the whole system to be deployed to succeed.
- It doesn't import anything from the datalake directory.